witwidget/notebook/jupyter/js/lib/wit.js

```diff
@@ -57,18 +57,10 @@ var WITView = widgets.DOMWidgetView.extend({
       witHtmlLocation = window.__nbextension_path__ + 'wit_jupyter.html';
     }
 
-    const src = `<link rel="import" href="${witHtmlLocation}">
-    <wit-dashboard local id="wit"></wit-dashboard>
```

Comment:
I think this change is related to the one in wit.html. Can you just comment on what this line is moved to wit.html?

Reply:
discussed offline